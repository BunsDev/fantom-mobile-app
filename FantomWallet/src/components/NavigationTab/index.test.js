import React from 'react';
import { shallow } from 'enzyme';

import Tab from './index';

const navigation = {
  state: {
    index: 5,
  },
};

test('renders correctly', () => {
  const wrapper = shallow(<Tab navigation={navigation} />);
  expect(wrapper).toMatchSnapshot();
});
