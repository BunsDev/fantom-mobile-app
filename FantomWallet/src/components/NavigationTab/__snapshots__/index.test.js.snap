// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "black",
      "bottom": 0,
      "elevation": 12,
      "flexDirection": "row",
      "position": "absolute",
      "shadowColor": "black",
      "shadowOffset": Object {
        "height": -5,
        "width": 0,
      },
      "shadowOpacity": 0.1,
      "width": 750,
    }
  }
>
  <Tab
    activeTabColor="rgb(0,177,251)"
    activeTabIndex={5}
    handleSelectedTab={[Function]}
    inActiveTabColor="rgb(38,44,50)"
    index={0}
    key="0"
    tabIfo={
      Object {
        "icon": Object {
          "testUri": "../../../src/images/WalletFilled.png",
        },
        "route": "Wallet",
      }
    }
  />
  <Tab
    activeTabColor="rgb(0,177,251)"
    activeTabIndex={5}
    handleSelectedTab={[Function]}
    inActiveTabColor="rgb(38,44,50)"
    index={1}
    key="1"
    tabIfo={
      Object {
        "icon": Object {
          "testUri": "../../../src/images/sendWhite.png",
        },
        "route": "Withdraw",
      }
    }
  />
  <Tab
    activeTabColor="rgb(0,177,251)"
    activeTabIndex={5}
    handleSelectedTab={[Function]}
    inActiveTabColor="rgb(38,44,50)"
    index={2}
    key="2"
    tabIfo={
      Object {
        "icon": Object {
          "testUri": "../../../src/images/downloading_white.png",
        },
        "route": "Deposit",
      }
    }
  />
</View>
`;
